### ISSUE TITLE
Publish under dual Unicode AND Open License

### ISSUE THREAD

**Issue by hugolpz** [unicode-org/unilex#10]
Your data is an impressive work which could help many, many minority and rare languages to get stronger online representation.

The Wikimedia Foundation, Wikipedia, Wikidata movements would love to use your data.

As far as I can see, your data is fully copyrighted but also release core rights :

> The Unicode Consortium releases this data under the same [license](http://unicode.org/copyright.html#License) as all its other data files. 
> Copyright © 1991-2020 Unicode, Inc. All rights 

And the [LICENSE.md](https://github.com/unicode-org/unilex/blob/master/LICENSE.md) file states :
> Copyright © 1991-2017 Unicode, Inc. All rights reserved. Distributed under the Terms of Use in http://www.unicode.org/copyright.html.
> Permission is hereby granted, free of charge, to any person obtaining a copy of the Unicode data files and any associated documentation (the “Data Files”) or Unicode software and any associated documentation (the “Software”) to deal in the Data Files or Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, and/or sell copies of the Data Files or Software, and to permit persons to whom the Data Files or Software are furnished to do so, provided that either (a) this copyright and permission notice appear with all copies of the Data Files or Software, or (b) this copyright and permission notice appear in associated Documentation.

This document mirrors the GNU license. The GNU license which is designed for softwares have been dropped by the Wikimedia community precisely because it ask to attach a page-long GNU document to all creations under GNU license. After long discussion, the Wikimedia Foundation decided to switch to Creative Commons License, which only require to attach a acronym such as `CCBYSA-4.0 + source` to the created product.

Could you add a dual (second) `CCBYSA-4.0` License to your License.md. The `CCBYSA-4.0` license mirror your current document agreements, but in a more shareable way.

**Comment by brawer** [comment]
For use by Wikimedia, [CC0-1.0](https://spdx.org/licenses/CC0-1.0.html) would actually be more useful than [CC-BY-4.0](https://spdx.org/licenses/CC-BY-4.0.html). In particular, [Wikidata Lexemes](https://www.wikidata.org/wiki/Wikidata:Lexicographical_data/Documentation) are licensed under CC0-1.0, not CC-BY-4.0. But that’s a minor detail.

@macchiati, what do you think? Would the Unicode consortium be open to dual-licensing Unilex under the Unicode Data License + Creative Commons Zero? Personally I think this would make the data more useful, but I wouldn’t know whom to ask to make it happen.

**Comment by macchiati** [comment]
Thanks for the head's up. Let me talk to our lawyer about that.

**Comment by hugolpz** [comment]
I tried to replicate your Unicode license. But Yes.
Wikidata will **love it** if you publish under CC0, which is like Public Domain.
If your objective is impact and to support language diversity, go for it. That's what we do ourselves (Wikimedia).

FYI / Info sharing :
Wikidata lexeme advancement is there : https://ordia.toolforge.org/language/ .
As far as I know, Wikidata Lexeme needs the triad `language + form + POS` and then can create a lexeme.
We still have to catch up with _state of the art_ lists such as [French's Lexique 3.83](https://github.com/chrplr/openlexicon) and others, but we will get there.

**Comment by macchiati** [comment]
Unfortunately, our lawyer is swamped now (and will continue to be in the near future), so I can't say when we would be able to get to this.

**Comment by hugolpz** [comment]
Thank you for this push. License switching within institutions is a marathon. This is one more positive wave. 👍🏼

**Comment by hugolpz** [comment]
@macchiati, @brawer : I think we can rename this issue as "Request open license", consider the job done, and close this issue. We will return to it when the lawyer give some feedbacks. I don't see any point to keep a zombie issue here.

**Comment by brawer** [comment]
We’re working on it. Let’s keep the bug open until either the license has been changed, or the request has officially been declined.

**Comment by srl295** [comment]
@hugolpz As I understand it Unicode artifacts (including ICU and CLDR) have already been used by Wikimedia for quite some time with the ICU and now Unicode license.  Unilex is now updated with the current (v3) Unicode license which is [OSI approved](https://opensource.org/license/unicode-license-v3/) as an open license.

@nemobis (if that's the right handle) do you have a comment from the Unicode/WMF side?

**Comment by nemobis** [comment]
@srl295 Thanks for the ping. I wasn't aware of this issue but I'll give a quick reply. I've only read the discussion above and the README. I can't speak for WMF, let alone Unicode (I don't remember whether WMF is even a member now), but I can tell about the usage of Unicode components in MediaWiki software and Wikimedia wikis.

The issue description highlights some confusion on the licensing of this project. Meanwhile the LICENSE has been updated to the Unicode license v3 which has been recently approved by OSI on 2023-11-17: https://opensource.org/license/unicode-license-v3/ . So there's no doubt this repository is opensource. Maybe this can be explicitly mentioned on the README, as not everyone is able to recognize the license text as its own OSI-approved Unicode v3 license.

MediaWiki can and does use software under Unicode license all the time, for example in the [CLDR extension](https://www.mediawiki.org/wiki/Extension:CLDR), which is primarily GPLv2, under the understanding that the CLDR data inside was under a BSD-like license. (Apertium linguistic data is also [usually](https://wiki.apertium.org/wiki/Contributing_to_an_existing_pair#Consider_contributing_your_improved_lexical_data) under GPL.) As long as Unilex can be used in GPL software, there are probably ways it can benefit all Wikimedia wikis through MediaWiki.

However @hugolpz seems most concerned about usage in Wikidata and other Wikimedia wikis _content_. From the README it sounds like this repository mostly wants to collect uncopyrightable factual information. In the EU, there might still be problems with database rights. A general opinion from the WMF on how to handle these is at https://meta.wikimedia.org/wiki/Wikilegal/Database_Rights . In short, it's complicated, and it's easier to incorporate a dataset into Wikidata when it's already under CC-0. If there's some doubt on whether the data/ directory here as a whole is a dataset 

If you want to cooperate with Wikidata lexemes in the future, it's worth considering how to make it easier. As for LinguaLibre, as far I understand it helps produce some recording which might be considered copyrightable, and it wants its outputs to be available under CC BY-SA, so it benefits from its sources being as permissive as possible.

Finally, I see that [many files](https://github.com/search?q=repo%3Aunicode-org%2Funilex+SPDX-License-Identifier&type=code) carry a `SPDX-License-Identifier: Unicode-DFS-2016` header, which makes it easier to follow the [Reuse](https://reuse.software/) guidelines. Note Richard Fontana's suggestion for trivial files at https://github.com/fsfe/reuse-docs/issues/62#issuecomment-1200305896 (and my personal opinion below it).

So in conclusion my personal suggestions are:
* mark the repository even more clearly as being under OSI-approved license Unicode v3;
* keep marking the individual files copyright status, and consider even more permissive licenses like MIT-0 (or 0BSD or CC-0) when adding uncopyrightable files;
* keep in mind possible copyright needs for Wikidata and Wikimedia Commons in the future, and ask help from WMF legal (legal@wikimedia.org) on any possible/needed clarifications for CC-0 and CC BY-SA compatibility (fyi @slaporte).

**Comment by annebright** [comment]
To add to Steven's comments, and to clarify, the Unicode License v3 (as well as the 2016 version) is based not on GNU but on the MIT license, which is about as open as it gets. The only difference between the MIT License and Unicode License is that the Unicode License expressly covers data files, which we see as an improvement over MIT, and presumably helpful to WikiMedia. We will not consider CC-0 or some other public domain declaration because we do not necessarily have the inbound license rights to do that.

**Comment by hugolpz** [comment]
For your information, there is a discussion to integrate Unilex data (new Unicode license) to Commons Wikimedia. It's a bit early, but it seems going in a positive way.
- https://commons.wikimedia.org/wiki/Commons:Village_pump/Copyright#Request_for_comments_:_Unicode_licensed_data
